connector/connect/common/src/main/protobuf/spark/connect/base.proto

```diff
@@ -304,8 +304,11 @@ message ExecutePlanResponse {
     // Response for commands on a streaming query.
     StreamingQueryCommandResult streaming_query_command_result = 9;
 
+    // Response for commands on the streaming query manager.
+    StreamingQueryManagerCommandResult streaming_query_manager_command_result = 10;
+
     // Response for 'SparkContext.resources'.
-    GetResourcesCommandResult get_resources_command_result = 10;
+    GetResourcesCommandResult get_resources_command_result = 11;
```

Comment:
This is a breaking change... 

probably you just do `StreamingQueryManagerCommandResult streaming_query_manager_command_result = 11;` and do not re-use existing proto number

Reply:
I see. TY i'll change it back
Just wanted to make similar names closer...